### PR TITLE
Add a order property to LinearPDESystemOperator

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

File: sumpy/expansion/diff_op.py [hunk 1]
```diff
@@ -65,6 +65,13 @@ def __eq__(self, other):
     def __hash__(self):
         return hash((self.dim, self.eqs))
 
+    @property
+    def degree(self):
```
Comment (inducer) [inline sumpy/expansion/diff_op.py:69]: I think "order" is the more conventional term. Would that work for you?

Comment (isuruf) [reply sumpy/expansion/diff_op.py]: Yes, works for me.

File: test/test_misc.py [hunk 2]
```diff
@@ -411,8 +411,11 @@ def test_as_scalar_pde_elasticity():
     ]
 
     pde = concat(*exprs)
+    assert pde.degree == 1
```
Comment (inducer) [inline test/test_misc.py:414]:
```suggestion
    assert pde.order == 1
```

File: test/test_misc.py [hunk 3]
```diff
@@ -411,8 +411,11 @@ def test_as_scalar_pde_elasticity():
...
     for i in range(5):
-        assert as_scalar_pde(pde, i) == laplacian(laplacian(diff_op[0]))
+        scalar_pde = as_scalar_pde(pde, i)
+        assert scalar_pde == laplacian(laplacian(diff_op[0]))
+        assert scalar_pde.degree == 4
```
Comment (inducer) [inline test/test_misc.py:418]:
```suggestion
        assert scalar_pde.order == 4
```

**Comment by inducer** [comment]
Thanks!

File: test/test_misc.py (hunk 3 above)
Comment (isuruf) [reply test/test_misc.py]: Thanks. Forgot to push this change.

**Pull request merged**